backend/graph/api/schema/schema.graphqls

```diff
@@ -127,4 +124,6 @@ type MutationRoot {
   setEpisodeProgress(id: ID!, progress: Int, duration: Int, context: EpisodeContext): Episode!
 
   sendSupportEmail(title: String!, content: String!, html: String!): Boolean!
+
+  completeTask(id: ID!, answer: TaskAnswer): Task!
```

Comment:
Should it maybe be 
```suggestion
  completeTask(id: ID!, answer: [TaskAnswer!]!): Task!
```
to support multi choice answers?

Reply:
Removed the answer field, as we aren't interested in the answers themselves at the moment, but I was thinking just a simple comma-separated answers whenever it was multiple select.